python_modules/libraries/dagster-twilio/dagster_twilio/resources.py

```diff
@@ -1,13 +1,23 @@
-from dagster import Field, StringSource, resource
+from dagster import ConfigurableResource, resource
+from dagster._config.structured_config import infer_schema_from_config_class
+from dagster._core.execution.context.init import InitResourceContext
+from pydantic import Field as PyField
```

Comment:
same question on the import as https://github.com/dagster-io/dagster/pull/12997#discussion_r1157777213

Reply:
def agree with @jamiedemaria that this aliasing is not helpful.